project-api/ballerina-projects/src/test/java/io/ballerina/projects/test/TestJarWriter.java

```diff
@@ -0,0 +1,64 @@
+/*
+ *  Copyright (c) 2020, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ *  WSO2 Inc. licenses this file to you under the Apache License,
+ *  Version 2.0 (the "License"); you may not use this file except
+ *  in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing,
+ *  software distributed under the License is distributed on an
+ *  "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ *  KIND, either express or implied.  See the License for the
+ *  specific language governing permissions and limitations
+ *  under the License.
+ */
+package io.ballerina.projects.test;
+
+import io.ballerina.projects.JarWriter;
+import io.ballerina.projects.Module;
+import io.ballerina.projects.Package;
+import io.ballerina.projects.directory.BuildProject;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+
+/**
+ * Contains cases to test the JarWriter.
+ *
+ * @since 2.0.0
+ */
+public class TestJarWriter {
+    private static final Path RESOURCE_DIRECTORY = Paths.get("src/test/resources/");
+    private final String dummyContent = "function foo() {\n}";
```

Comment:
`dummyContent` not used anywhere, Are we hoping to use it in the future?

Reply:
No, we can remove it. Fixed in d50564d